src/Transactions/TransactionListFromBudget/TransactionListFromBudget.js

```diff
@@ -0,0 +1,24 @@
+import React, { useCallback } from 'react';
+import { withRouter } from 'react-router-dom';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import TransactionsListContainer from '../TransactionsList';
+import { BUDGET_ROUTE } from '../../common/const';
+
+const TransactionsListFromBudget = ({ match, history }) => {
```

Comment:
IMHO, it's better to use anonymous components, since the only thing there is to pass a proper `closePane`